be/src/main/java/yeonba/be/login/controller/LoginController.java

```diff
@@ -45,6 +45,31 @@ public ResponseEntity<CustomResponse<UserJoinResponse>> join(
             .body(new CustomResponse<>(response));
     }
 
+    @Operation(summary = "소셜 로그인", description = "소셜 로그인을 할 수 있습니다.")
+    @ApiResponse(responseCode = "200", description = "로그인 성공")
+    @PostMapping("/users/login")
+    public ResponseEntity<CustomResponse<UserLoginResponse>> login(
+        @RequestBody UserLoginRequest request) {
+
+        UserLoginResponse response = loginService.login(request);
+
+        return ResponseEntity
+            .ok()
+            .body(new CustomResponse<>(response));
+    }
+
+    @Operation(summary = "jwt 재발급", description = "refresh token을 통해 jwt를 재발급받을 수 있습니다.")
+    @PostMapping("/users/refresh")
```

Comment:
```suggestion
    @Operation(summary = "jwt 재발급", description = "refresh token을 통해 jwt를 재발급받을 수 있습니다.")
    @ApiResponse(responseCode="200", description="jwt 정상 재발급")
    @PostMapping("/users/refresh")
```
`@ApiResponse`추가 부탁드려요~